internal/cli/deployments/diagnostics.go

```diff
@@ -59,34 +64,51 @@ func (opts *diagnosticsOpts) Run(ctx context.Context) error {
 		Engine: opts.ContainerEngine.Name(),
 	}
 
-	images, err := opts.ContainerEngine.ImageList(ctx)
+	cores, err := cpu.Counts(true)
```

Comment:
do we call diagnostics on e2e test failures? would be a nice approach to ensure we call it in some tests.. overall I think as this command has grown, we should add unit/e2e tests for it, but can be in another ticket!

Reply:
Yes we do

Reply:
https://github.com/mongodb/mongodb-atlas-cli/blob/010d10c5fdcc06ff12f08e3b5d15189a2265e9ff/test/e2e/atlas/deployments_local_auth_test.go#L50-L62